src/lib/pp/ppCalculation.ts

```diff
@@ -0,0 +1,64 @@
+import { PPAdjustingMod } from "../modIcons";
+import { applyMods } from "../od/overallDifficulty";
+
+export function calcAcc(maxCombo: number, goods: number, misses: number) {
```

Comment:
I suggest moving this function out to `lib/osu-utils.ts` since this calculation could be reused elsewhere as well

Reply:
boom